src/Features/CSharp/Portable/ConvertBetweenRegularAndVerbatimString/AbstractConvertBetweenRegularAndVerbatimStringCodeRefactoringProvider.cs

```diff
@@ -53,20 +53,28 @@ public sealed override async Task ComputeRefactoringsAsync(CodeRefactoringContex
                     return;
             }
 
+            // Note: This is a generally useful feature on strings.  But it's not likely to be something
+            // people want to use a lot.  Make low priority so it doesn't interfere with more
+            // commonly useful refactorings.
+
             if (IsVerbatim(literalExpression))
             {
                 // always offer to convert from verbatim string to normal string.
-                context.RegisterRefactoring(new MyCodeAction(
+                context.RegisterRefactoring(CodeAction.CreateWithPriority(
```

Comment:
I added a new internal helper `CodeAction.CreateWithPriority` for creating code actions with non-default priority. We no longer need to create special code action sub-types just to override the priority.